strongbox-web-core/src/main/java/org/carlspring/strongbox/security/authentication/strategy/BasicAuthenticationStrategy.java

```diff
@@ -20,28 +20,23 @@
  */
 @Component
 @Order(4)
-class BasicAuthenticationSupplier implements AuthenticationSupplier
+class BasicAuthenticationStrategy implements AuthenticationStrategy
```

Comment:
+1 for the naming! :)

Reply:
not sure that `*AuthenticationStrategy` is good naming here, because this interface do not authenticate by itself, it just supplies the `Authentication` object based on `HttpServletRequest`.

Reply:
Sorry, I read it too fast before bed. You're right.